ietf-optical-impairment-topology.tree

```diff
@@ -14,10 +15,10 @@ module: ietf-optical-impairment-topology
        +--ro media-channel-group* [i]
        |  +--ro i                 int16
        |  +--ro media-channels* [flexi-n]
-       |     +--ro flexi-n      uint16
+       |     +--ro flexi-n      int16
```

Comment:
is the n and m granularity specified somewhere else ?

Reply:
yes, in layer0-types 
https://raw.githubusercontent.com/ietf-ccamp-wg/draft-ietf-ccamp-layer0-types/2bce4f072ee0ec23b1f2a2abf22c98a4877f7f4e/ietf-layer0-types.yang

Reply:
ok, then we should specify also the granularity here.  n and m do not have any meaning w/o knowing their granularity.